src/components/ConcertCard.js

```diff
@@ -55,8 +48,8 @@ const Wrapper = styled.section`
   letter-spacing: 0.1em;
   list-style: none;
   display: grid;
-  grid-template-columns: 75% 25%;
-  grid-template-rows: 7ch;
+  grid-template-columns: 70% 30%;
```

Comment:
This does not work with `gap`. 
```suggestion
  grid-template-columns: 7fr 3fr;
```